app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, file, file_to_copy = command.split()
+    if command_name == "cp" and file != file_to_copy:
+        try:
+            with (open(file, "r") as current_file,
+                  open(file_to_copy, "w") as new_file):
+                new_file.write(current_file.read())
+        except FileNotFoundError as found_error:
+            print(found_error)
+            return
```

Comment:
You catching all you need in the if condition in 3 line, you can skip try/except

Reply:
I remove it